src/Appwrite/Platform/Tasks/DeleteProjectCollections.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Utopia\App;
+use Utopia\Platform\Action;
+use Utopia\Cache\Cache;
+use Utopia\CLI\Console;
+use Utopia\Database\Database;
+use Utopia\Database\Query;
+use Utopia\Pools\Group;
+use Utopia\Registry\Registry;
+
+class DeleteProjectCollections extends Action
+{
+    private array $names = [
+        'webhooks',
+        'platforms',
+        'schedules',
+        'projects',
+        'domains',
+        'certificates',
+        'keys',
+        'realtime',
```

Comment:
How come we don't need these?

Reply:
These are not used on the project DB. They are only required for the consoleDB

Reply:
However they were created anyways since they're all part of the same collections.php file and we didnt distinguish between console and project tables so far